components/language_filter/src/main.py

```diff
@@ -0,0 +1,70 @@
+"""A component that filter a provided dataframe based on the language"""
+import logging
+import dask.dataframe as dd
+from fondant.component import DaskTransformComponent
+from fondant.logger import configure_logging
+import fasttext
+
+configure_logging()
+logger = logging.getLogger(__name__)
+
+
+class LanguageIdentification:
+    """A class for language detection using FastText."""
+
+    def __init__(self, model_path: str = "lid.176.ftz"):
+        """
+       Initializes the LanguageDetect class.
+
+       Args:
+           model_path (str): The path to the FastText language identification model.
+       """
+        pretrained_lang_model_weight_path = model_path
+        self.model = fasttext.load_model(pretrained_lang_model_weight_path)
+
+    def predict_lang(self, text: str):
+        """
+        Detects the language of a text sequence.
+
+        Args:
+            text (str): The text for language detection.
+
+        Returns:
+            str: The predicted language label.
+        """
+        predictions = self.model.predict(text, k=1)
+        return predictions[0][0]
+
+    def is_language(self, row, language):
+        return language in self.predict_lang(row["text"])
+
+
+class LanguageFilterComponent(DaskTransformComponent):
```

Comment:
```suggestion
class LanguageFilterComponent(PandasTransformComponent):
```
Feel free to upgrade to the Pandas interface (see caption_images as an example)

Reply:
Where is the difference between them? Isn't `DaskTransformComponent` the default way for transformation?